docs/setup-cluster/basic-setup-guide/basic.rst

```diff
@@ -0,0 +1,56 @@
+.. _basic-setup:
+
+.. _install-cluster:
+
+#############
+ Basic Setup
+#############
+
+If you have not used Determined before, and you want to quickly set up a new training environment,
+you are at the right place!
+
+.. note::
+
+   To set up a Determined training environment on-prem or on cloud, visit :ref:`Advanced Setup
+   <advanced-setup>`.
+
+
+**Prerequisites**
+
+-  :ref:`Installation Requirements <system-requirements>`
+
+*********************************
+ Step 1 - Install Docker Desktop
+*********************************
+
+-  :ref:`Install Docker <install-docker>` on your machine.
```

Comment:
linking to another doc is ok, but if there was a good time to use tabs, this is probably a good one.  It's nice if the basic setup guide is all-in-one.

Reply:
I consider this resolved.

The "Install Docker" page itself is too complex to be a tab within this doc but I did simplify the Install Docker page by applying tabbed content to the first two steps.